packages/styled-react/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import {Tooltip as PrimerTooltip, type TooltipProps as PrimerTooltipProps, type SxProp} from '@primer/react'
+import {
+  Tooltip as PrimerDeprecatedTooltip,
+  type TooltipProps as PrimerDeprecatedTooltipProps,
+} from '@primer/react/deprecated'
+import {Box} from './Box'
+import {forwardRef, type ForwardRefExoticComponent, type RefAttributes} from 'react'
+
+type TooltipProps = PrimerTooltipProps & SxProp
+
+const Tooltip: ForwardRefExoticComponent<TooltipProps & RefAttributes<HTMLDivElement>> = forwardRef<
+  HTMLDivElement,
+  TooltipProps
+>(function Tooltip(props, ref) {
+  return <Box as={PrimerTooltip} ref={ref} {...props} />
+})
+
+export {Tooltip, type TooltipProps}
+
+type DeprecatedTooltipProps = PrimerDeprecatedTooltipProps & SxProp
+
+function DeprecatedTooltip(props: DeprecatedTooltipProps) {
+  return <Box as={PrimerDeprecatedTooltip} {...props} />
+}
```

Comment:
should we do the new `as` hack just in case? I think in theory it affects every component

Reply:
I think this only affects `polymorphic` components that take an `as` prop, `Tooltip` isn’t one of them, so it should be fine.
I tested it in `primer-react.browser.test.tsx` and confirmed that the props and children are passed correctly for `Tooltip`.

Reply:
ah got it, thanks! ❤️